src/snowflake/cli/plugins/cortex/commands.py

```diff
@@ -1,6 +1,257 @@
+        )
+
+    return MessageResult(result_text.strip())
+
+
+@app.command(
+    name="sentiment",
+    requires_connection=True,
+)
+def sentiment(
+    text: Optional[str] = typer.Argument(
+        None,
+        help="String containing the text for which a sentiment score should be calculated.",
+        show_default=False,
+    ),
+    file: Optional[Path] = readable_file_option(
+        param_name="--file",
+        help_str="File containing the text for which a sentiment score should be calculated.",
+    ),
+    **options,
+) -> CommandResult:
+    """
+    Returns sentiment as a score between -1 to 1
+    (with -1 being the most negative and 1 the most positive,
+    with values around 0 neutral) for the given English-language input text.
+    """
+
+    manager = CortexManager()
+
+    if text and file:
+        raise UsageError("--file option cannot be used together with TEXT argument.")
+    if text:
+        result_text = manager.calculate_sentiment_for_text(
+            text=Text(text),
+        )
+    elif file:
+        result_text = manager.calculate_sentiment_for_text_file(
+            text_file=SecurePath(file),
+        )
+    else:
+        raise UsageError("Either --file option or TEXT argument has to be provided.")
+
+    return MessageResult(result_text.strip())
+
+
+@app.command(
+    name="summarize",
+    requires_connection=True,
+)
+def summarize(
+    text: Optional[str] = typer.Argument(
+        None,
+        help="String containing the English text from which a summary should be generated.",
+        show_default=False,
+    ),
+    file: Optional[Path] = readable_file_option(
+        param_name="--file",
+        help_str="File containing the English text from which a summary should be generated.",
+    ),
+    **options,
+) -> CommandResult:
+    """
+    Summarizes the given English-language input text.
+    """
+
+    manager = CortexManager()
+
+    if text and file:
+        raise UsageError("--file option cannot be used together with TEXT argument.")
+    if text:
+        result_text = manager.summarize_text(
+            text=Text(text),
+        )
+    elif file:
+        result_text = manager.summarize_text_file(
+            text_file=SecurePath(file),
+        )
+    else:
+        raise UsageError("Either --file option or TEXT argument has to be provided.")
+
+    return MessageResult(result_text.strip())
+
+
+@app.command(
+    name="translate",
+    requires_connection=True,
+)
+def translate(
+    text: Optional[str] = typer.Argument(
+        None,
+        help="String containing the text to be translated.",
+        show_default=False,
+    ),
+    from_language: Optional[str] = typer.Option(
+        None,
+        "--from",
+        help="String specifying the language code for the language the text is currently in. See Snowflake Cortex documentation for a list of supported language codes.",
+        show_default=False,
+    ),
+    to_language: str = typer.Option(
+        ...,
```

Comment:
Should we use `en` as default?

Reply:
I don't think so. The doc with proposal didn't present any default for target language, same as TRANSLATE function doesn't provide any default (https://docs.snowflake.com/en/sql-reference/functions/translate-snowflake-cortex)

Reply:
But most of the users are english-speaking. Not having to specify `--to=en` every time sounds like a good thing to have.

Reply:
Discussed and agreed to keep it without a default value. Removing a default value in the future will be backward incompatible while adding it if needed will a simple compatible change.